roles/openshift_master/tasks/main.yml

```diff
@@ -77,6 +81,25 @@
 - name: Install Master package
   yum: pkg={{ openshift.common.service_type }}-master{{ openshift_version  }} state=present
   register: install_result
+  when: not is_containerized | bool
+
+# TODO: enable when ansible#1993 lands and is widespread enough
+# - name: Docker image present
+#   docker:
+#     image: "{{ openshift.common.docker.image }}"
+#     state: image_present
+#   when: is_containerized | bool
```

Comment:
I'm wondering if we need to break down some of these roles into separate task files that are included conditionally based on different attributes. That way we could still de-duplicate common functionality, but ha vs non-ha, containerized vs non containerized might be more straight forward to reason when reading.